data/sql_updates/create_candidate_detail_view.sql

```diff
@@ -1,5 +1,6 @@
-drop materialized view if exists ofec_candidate_detail_vw;
-create materialized view ofec_candidate_detail_vw as
+drop view if exists ofec_candidate_detail_vw;
```

Comment:
This is already a materialized view so this line will error out. This was the weird problem we were having last week.